Makefile

```diff
@@ -114,7 +114,13 @@ run-integ-test:
 	# Also adding count=1 so the test results aren't cached.
 	# This command also targets files with the build integration tag
 	# and runs tests which end in Integration.
-	go test -race -count=1 -timeout 60m -tags=integration ${PACKAGES}
+	go test -race -count=1 -timeout 60m -tags=integration,integration-remote ${PACKAGES}
```

Comment:
do we  need to specify `integration-remote` here or can we just have `-tags=integration`

Reply:
hmm, maybe you're right--let me check